### PR TITLE
tests: sundials/harmonic_oscillator_04 increase precision

### LINKED ISSUE #5497: sundials/harmonic_oscillator_04 test is still broken on most testers
https://github.com/dealii/dealii/pull/5457 did not help 
https://cdash.kyomu.43-1.org/testSummary.php?project=1&name=sundials%2Fharmonic_oscillator_04.release&date=2017-11-17

@luca-heltai you know what's going on?

### PR DESCRIPTION
**Pull request by tamiko**
Fixes #5497.

edit (masterleinad): Add a reference to the respective issue.

### CODE REVIEW

File: tests/sundials/harmonic_oscillator_04.output [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+1 2.075274578 1.059601971 2.499942015
+2 1.100460018 1.723610112 2.499960048
+3 0.7896202372 2.326668043 2.499977831
+4 0.8117726419 2.725583967 2.499921064
+5 1.18700292 2.595541877 2.500052271
+6 1.887311245 1.494098161 2.499912467
+7 1.323271336 1.619016375 2.499926279
+8 0.9217874621 2.13021939 2.49998867
+9 0.8495425448 2.539475112 2.500048538
+10 1.064969837 2.595955796 2.499973862
```
Comment (tamiko) [inline tests/sundials/harmonic_oscillator_04.output]: @luca-heltai *ping* Are such large deviations expected?

**Comment by tamiko** [comment]
This should do the trick. :-)

**Comment by masterleinad** [comment]
> This should do the trick. :-)

Thanks! So the parameter file was the problem here in the end.

**Comment by masterleinad** [comment]
I took the freedom to reference the respective issue.

**Pull request merged**